pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -144,6 +144,9 @@ var (
 		}, {
 			Name:  "SERVING_REQUEST_METRICS_BACKEND",
 			Value: "",
+		}, {
+			Name:  "SERVING_REQUEST_METRICS_REPORTING_PERIOD_SECONDS",
+			Value: "0",
```

Comment:
shouldn't this be a default value?

Reply:
zero or negative value is translated by opencensus to the default value which is 10s. Here we have the default of 5sec for Prometheus and 60sec for Opencensus.